src/server.ts

```
@@ -61,8 +62,13 @@
     case 'health_status':
       return reply.code(200).send({result: 'ok'});
     case 'sign':
-        const signature = await wallets.ecsign({keyId: keyId}, Buffer.from(params[0].slice(2), 'hex'), 1);
-        const res = Buffer.concat([signature.r, signature.s, bigintToBuffer(signature.v)]).toString('hex');
+      let digest = Buffer.from(params[0].slice(2), 'hex');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)